Woof/Woof/View/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+/// A view displaying detailed information about the sitter.
+struct DetailPetSitterView: View {
+    /// The full name of the sitter.
+    let fullName: String
+    /// The phone number of the sitter.
+    let phoneNumber: String
```

Comment:
Do we need this?
The view model has to provide this data.

Reply:
Yes, we require an object that can provide data for viewing purposes.
``` swift
@ObservedObject var viewModel: DetailSitterViewModel
```

Reply:
Thanks, fixed.